frontend/src/views/shell/shell.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
Can I move this to the UI folder?

Reply:
Why do you think it's better for the Shell to be in the ui?
My thinking is that `ui/*` is for ui components, which should work independently as much as possible.
The Shell is an application view component, which refers all the other subpages of the app.

Reply:
My understanding of the "view" folder should be components associated to the applications routes. In our example we have two routes (players and games), therefore we will have two "views" of that routes. For me the shell works as an ui component which can reused in multiple views.

Reply:
The landing page could have its own view potentially as well